2023-03-28 00:06:51,293 [1] INFO  FantasyOfSango.SangoServer - Sango Server is Running
2023-03-28 00:06:51,309 [1] INFO  FantasyOfSango.SangoServer - Init all Component
2023-03-28 00:06:51,310 [1] INFO  Photon.SocketServer.ApplicationBase - Application start: AppId=SangoTest; AppPath=D:\FantasyOfSango\Server\TestServer\Photon_v4\FantasyOfSango_Photon\deploy\FantasyOfSango, Type=FantasyOfSango.SangoServer 
2023-03-28 00:06:56,216 [11] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 00:07:11,464 [10] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:0,ip:127.0.0.1:62219, Data: F3-02-07-00-01-08-73-01-1E-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-31-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-38-2E-32-33-31-38-33-34-34-2C-22-59-22-3A-35-2E-31-37-32-33-39-37-36-31-2C-22-5A-22-3A-31-32-30-2E-35-39-39-32-38-39-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-39-2E-32-35-38-33-35-34-32-2C-22-59-22-3A-35-2E-37-39-35-37-30-30-35-35-2C-22-5A-22-3A-31-32-32-2E-39-30-35-34-34-39-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-32-37-54-31-36-3A-30-37-3A-31-31-2E-33-37-31-39-37-34-31-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-28 00:07:18,133 [14] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 01:54:16,318 [1] INFO  Photon.SocketServer.ApplicationBase - Application is stopping: AppId=SangoTest
2023-03-28 01:54:16,325 [16] ERROR Photon.SocketServer.ApplicationBase - System.Threading.ThreadAbortException: Thread was being aborted.
   at System.Threading.Thread.SleepInternal(Int32 millisecondsTimeout)
   at System.Threading.Thread.Sleep(Int32 millisecondsTimeout)
   at FantasyOfSango.Threads.SyncPlayerTransformThreads.Update() in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Threads\SyncPlayerTransformThreads.cs:line 27
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
2023-03-28 01:54:16,340 [1] INFO  FantasyOfSango.SangoServer - Sango Server is Shut Down
2023-03-28 01:54:16,340 [1] INFO  Photon.SocketServer.ApplicationBase - Application stop: AppId=SangoTest
2023-03-28 14:32:29,509 [1] INFO  FantasyOfSango.SangoServer - Sango Server is Running
2023-03-28 14:32:29,539 [1] INFO  FantasyOfSango.SangoServer - Init all Component
2023-03-28 14:32:29,539 [1] INFO  Photon.SocketServer.ApplicationBase - Application start: AppId=SangoTest; AppPath=D:\FantasyOfSango\Server\TestServer\Photon_v4\FantasyOfSango_Photon\deploy\FantasyOfSango, Type=FantasyOfSango.SangoServer 
2023-03-28 14:32:33,200 [11] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 14:39:13,835 [23] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:0,ip:127.0.0.1:59407, Data: F3-02-07-00-01-08-73-01-1E-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-31-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-37-2E-34-34-36-34-37-39-38-2C-22-59-22-3A-35-2E-30-32-30-33-38-33-38-33-2C-22-5A-22-3A-31-31-39-2E-38-32-33-33-30-33-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-39-2E-32-35-38-33-35-34-32-2C-22-59-22-3A-35-2E-37-39-35-37-30-30-35-35-2C-22-5A-22-3A-31-32-32-2E-39-30-35-34-34-39-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-32-38-54-30-36-3A-33-39-3A-31-33-2E-37-34-37-36-32-34-32-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-28 14:39:24,499 [18] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:0,ip:127.0.0.1:59407, Data: F3-02-07-00-01-08-73-01-1C-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-31-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-35-31-2E-30-34-36-35-38-35-31-2C-22-59-22-3A-35-2E-36-32-30-39-32-39-37-32-2C-22-5A-22-3A-31-32-33-2E-32-30-38-31-33-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-39-2E-32-35-38-33-35-34-32-2C-22-59-22-3A-35-2E-37-39-35-36-39-39-36-2C-22-5A-22-3A-31-32-32-2E-39-30-35-34-34-39-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-32-38-54-30-36-3A-33-39-3A-32-34-2E-34-36-30-32-37-33-38-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-28 14:39:40,948 [14] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 14:57:48,651 [14] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 14:58:31,827 [18] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:1,ip:127.0.0.1:53752, Data: F3-02-07-00-01-08-73-01-1F-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-31-30-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-37-2E-38-37-39-30-37-34-31-2C-22-59-22-3A-35-2E-39-37-38-33-39-35-34-36-2C-22-5A-22-3A-31-32-33-2E-34-31-32-35-36-37-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-39-2E-32-35-38-33-35-34-32-2C-22-59-22-3A-35-2E-37-39-35-37-30-30-30-37-2C-22-5A-22-3A-31-32-32-2E-39-30-35-34-34-39-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-32-38-54-30-36-3A-35-38-3A-33-31-2E-37-38-34-35-38-32-38-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-28 14:58:34,308 [13] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:1,ip:127.0.0.1:53752, Data: F3-02-07-00-01-08-73-01-1D-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-31-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-37-2E-38-37-39-30-37-34-31-2C-22-59-22-3A-35-2E-39-37-38-33-39-35-34-36-2C-22-5A-22-3A-31-32-33-2E-34-31-32-35-36-37-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-39-2E-32-35-38-33-35-34-32-2C-22-59-22-3A-35-2E-37-39-35-36-39-39-36-2C-22-5A-22-3A-31-32-32-2E-39-30-35-34-34-39-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-32-38-54-30-36-3A-35-38-3A-33-34-2E-32-36-36-31-37-38-32-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-28 14:58:43,046 [22] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:1,ip:127.0.0.1:53752, Data: F3-02-07-00-01-08-73-01-1D-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-38-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-37-2E-37-30-35-31-35-34-34-2C-22-59-22-3A-36-2E-30-38-32-39-32-35-38-2C-22-5A-22-3A-31-32-33-2E-36-38-31-31-36-38-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-39-2E-32-35-38-33-35-34-32-2C-22-59-22-3A-35-2E-37-39-35-37-30-30-30-37-2C-22-5A-22-3A-31-32-32-2E-39-30-35-34-34-39-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-32-38-54-30-36-3A-35-38-3A-34-33-2E-30-30-38-33-30-31-37-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-28 14:58:50,000 [10] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 20:13:42,935 [14] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 20:17:47,377 [20] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 20:22:34,306 [19] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 20:23:28,214 [11] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 20:24:36,129 [15] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 20:32:19,661 [15] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 20:33:40,690 [13] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 20:35:06,078 [15] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 21:28:17,198 [20] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 21:28:41,084 [19] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:6,ip:127.0.0.1:56508, Data: F3-02-07-00-01-08-73-01-25-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-28-43-6C-6F-6E-65-29-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-31-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-36-38-2E-33-30-34-35-37-33-31-2C-22-59-22-3A-31-31-2E-34-30-30-30-35-38-37-2C-22-5A-22-3A-31-34-36-2E-37-33-38-39-36-38-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-36-39-2E-38-38-39-38-31-36-33-2C-22-59-22-3A-31-31-2E-38-34-37-31-33-31-37-2C-22-5A-22-3A-31-34-38-2E-39-36-37-36-35-31-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-32-38-54-31-33-3A-32-38-3A-34-30-2E-39-30-30-34-32-31-32-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-28 21:29:09,834 [23] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 21:35:43,109 [19] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 21:36:09,357 [11] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:7,ip:127.0.0.1:54572, Data: F3-02-07-00-01-08-73-01-23-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-28-43-6C-6F-6E-65-29-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-31-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-37-2E-31-39-31-30-34-2C-22-59-22-3A-36-2E-37-35-33-35-34-32-34-32-2C-22-5A-22-3A-31-32-35-2E-36-36-39-38-31-35-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-34-35-2E-37-31-38-33-30-37-35-2C-22-59-22-3A-37-2E-30-37-33-36-35-38-39-34-2C-22-5A-22-3A-31-32-36-2E-31-31-30-33-35-39-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-32-38-54-31-33-3A-33-36-3A-30-39-2E-32-39-35-38-35-31-33-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-28 21:36:13,530 [18] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 21:39:48,562 [14] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 21:40:17,143 [15] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 21:42:24,692 [20] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 21:42:55,297 [21] ERROR Photon.SocketServer.PeerBase - exception during handling of data.Peer:T:ClientPeer,ConnId:9,ip:127.0.0.1:51963, Data: F3-02-07-00-01-08-73-01-24-7B-22-46-69-67-68-74-54-79-70-65-43-6F-64-65-22-3A-30-2C-22-41-74-74-61-63-6B-65-72-41-63-63-6F-75-6E-74-22-3A-22-61-22-2C-22-44-61-6D-61-67-65-72-41-63-63-6F-75-6E-74-22-3A-22-4D-6F-6E-73-74-65-72-5F-48-69-6C-69-28-43-6C-6F-6E-65-29-22-2C-22-53-6B-69-6C-6C-43-6F-64-65-22-3A-35-2C-22-45-6C-65-6D-65-6E-74-52-65-61-63-74-69-6F-6E-43-6F-64-65-22-3A-31-2C-22-41-74-74-61-63-6B-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-35-31-2E-35-33-33-30-31-32-34-2C-22-59-22-3A-38-2E-35-30-37-39-31-31-36-38-2C-22-5A-22-3A-31-33-30-2E-35-39-36-37-31-7D-2C-22-44-61-6D-61-67-65-72-50-6F-73-69-74-69-6F-6E-22-3A-7B-22-58-22-3A-2D-35-31-2E-33-38-38-38-36-32-36-2C-22-59-22-3A-37-2E-37-34-39-36-37-31-39-34-2C-22-5A-22-3A-31-33-30-2E-35-31-31-30-33-32-7D-2C-22-44-61-74-65-54-69-6D-65-22-3A-22-32-30-32-33-2D-30-33-2D-32-38-54-31-33-3A-34-32-3A-35-35-2E-32-33-38-32-36-31-32-5A-22-7D,
Exception:System.NullReferenceException: Object reference not set to an instance of an object.
   at FantasyOfSango.System.OnlineAttackSystem.SetElementAttackDamage(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 76
   at FantasyOfSango.System.OnlineAttackSystem.GetAttackResult(AttackDamageCache attackDamageCache) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\System\OnlineAttackSystem.cs:line 31
   at FantasyOfSango.Handler.AttackDamageHandler.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters, ClientPeer peer) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\Handler\AttackDamageHandler.cs:line 22
   at FantasyOfSango.ClientPeer.OnOperationRequest(OperationRequest operationRequest, SendParameters sendParameters) in D:\FantasyOfSango\Server\TestServer\TestServerScript\FantasyOfSango\FantasyOfSango\ClientPeer.cs:line 37
   at Photon.SocketServer.PeerBase.OnReceive(Byte[] data, SendParameters sendParameters)
   at Photon.SocketServer.PeerBase.OnReceiveInternal(Byte[] data, SendParameters sendParameters, Int32 rtt, Int32 rttVariance, Int32 numFailures)
2023-03-28 21:43:01,180 [10] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 21:52:18,452 [21] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 21:52:56,211 [23] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 21:56:43,213 [10] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 21:57:27,822 [13] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 22:02:52,599 [23] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 22:05:31,020 [21] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 22:24:54,729 [15] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 22:25:38,920 [10] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 23:10:36,863 [12] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 23:11:04,319 [13] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 23:13:44,767 [14] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 23:14:12,176 [23] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
2023-03-28 23:14:28,800 [21] INFO  FantasyOfSango.SangoServer - New Client connected
2023-03-28 23:14:57,667 [12] INFO  FantasyOfSango.SangoServer - The Client is DisConnect
